src/controllers/ReportController.js

```diff
@@ -1,11 +1,66 @@
-import models from '../db/models';
 import Response from '../helpers/response';
+import models from '../db/models';
+import EmailNotificationAPI from '../helpers/EmailNotificationAPI';
 
-const { Report, Art, User } = models;
+const {
+  Report, Art, User, ReportSummary
+} = models;
+
+const reporterMessage = `<h1>Feed back </h1>
+<p> Thanks for reaching out to us on your consigns, 
+The necessary action has been taken on the report</p>
+<strong>Thank you for choosing Art Cave</strong>
+  `;
 
-/** Report Controller */
-class ReportController {
 /**
+ * Represents a ReportController.
+ */
+class ReportController {
+  /**
+   * @static
+   * @param {Object} req
+   * @param {object} res
+   * @return {object} report
+   */
+  static async deleteReport(req, res) {
+    try {
+      const { reportId, reporterId } = req.params;
+
+      await Report.destroy({
+        where: { id: reportId }
+      });
+      const reporter = await User.find({
```

Comment:
If the user must be authenticated before reporting a user/article, then you should be able to get the user's information from the `req`. I'm assuming you are attaching the user's information to the request after validating jwt.

Reply:
Yes, the user must be logged in but this is admin that is logged in here trying to perform some actions on reported article. So  What I did is to refactor it to query the report table and get the reporter Id instead of getting it from the params